domain/src/main/kotlin/no/nav/su/se/bakover/domain/visitor/LagBrevRequestVisitor.kt

```diff
@@ -968,13 +978,14 @@ class LagBrevRequestVisitor(
         gjeldendeMånedsutbetaling = gjeldendeMånedsutbetaling,
         dagensDato = LocalDate.now(clock),
         saksnummer = saksnummer,
-        satsoversikt = Satsoversikt.fra(bosituasjon),
+        satsoversikt = Satsoversikt.fra(bosituasjon, satsFactory),
     )
 }
 
 private fun Vilkår.Formue.hentFormueGrunnlagForSøknadsbehandling(avslagsgrunner: List<Avslagsgrunn>): Formuegrunnlag? {
     return when (this) {
-        Vilkår.Formue.IkkeVurdert -> null
+        is Vilkår.Formue.IkkeVurdert -> null
+        // TODO(satsfactory_formue) jah: jeg har ikke endret funksjonaliteten i Sats-omskrivningsrunden, men hvorfor sjekker vi avslagsgrunn for å avgjøre dette? De burde jo uansett henge sammen.
```

Comment:
Forsøk på å styre innhold i brevet kanskje.. eller kanskje bare en wrapper for å komme seg fra generell type til en spesifikk med garantert 1 grunnlag? Kan ikke se at vi viser noe knyttet til formue i brevet bortsett fra tilfeller hvor avslagsgrunn er formue.. disse skal gå hånd i hånd.